docs/fleet/provision.md

```diff
@@ -1,278 +1,186 @@
-If you are not using Flutter or TypeScript and would like to use provisioning, please [contact us](mailto:support@viam.com).
+### Attributes
+
+<!-- prettier-ignore -->
+| Name       | Type   | Required? | Description |
+| ---------- | ------ | --------- | ----------- |
+| `manufacturer` | string | Optional | Purely informative. May be displayed on captive portal and/or mobile app. Default: `"viam"`. |
+| `model` | string | Optional | Purely informative. May be displayed on captive portal and/or mobile app. Default: `"custom"`. |
+| `fragment_id` | string | Optional | The `fragment_id` of the fragment to configure machines with. If present, the Viam mobile app can pre-configure a robot for a user by using this [TODO: and otherwise it can't??]. |
+| `hotspot_prefix` | string | Optional | Viam agent will prepend this to the hostname of the device append and use the resulting string for the provisioning hotspot SSID. Default: `"viam-setup"`. |
+| `disable_dns_redirect` | boolean | Optional | By default, ALL DNS lookups using the provisioning hotspot will redirect to the device. This causes most phones/mobile devices to automatically redirect the user to the captive portal as a "sign in" screen. When disabled, only domains ending in .setup (ex: viam.setup) will be redirected. This generally avoids displaying the portal to users and is mainly used in conjunction with a mobile provisioning application workflow. Default: `false`. |
+| `hotspot_password` | string | Optional | The Wifi password for provisioning hotspot. Default: `"viamsetup"`. |
+| `roaming_mode` | boolean | Optional | By default, the device will only attempt to connect to a single wifi network (the one with the highest priority), usually provided during initial provisioning/setup using the Viam app or the Viam mobile app. Wifi connection alone is enough to consider the device as "online" even if the global internet is not reachable. When enabled, the device will attempt connections to all configured networks, and only considers the device online if the internet is reachable. See [roaming mode](#how-an-end-user-would-use-it). Default: `false`. |
+| `offline_timeout` | boolean | Optional | Will only enter provisioning mode (hotspot) after being disconnected longer than this time. Useful on flaky connections, or when part of a system where the device may start quickly, but the wifi/router may take longer to be available. Default: `"2m"` (2 minutes). |
+| `user_timeout` | boolean | Optional | Amount of time before considering a user (using the provisioning portal using web or mobile app) idle, and resuming normal behavior. Used to avoid interrupting provisioning mode (for example for network tests/retries) when a user might be busy entering details. Default: `"5m"` (5 minutes). |
```

Comment:
```suggestion
| `user_timeout` | boolean | Optional | Amount of time before considering a user (using the captive web portal or mobile app) idle, and resuming normal behavior. Used to avoid interrupting provisioning mode (for example for network tests/retries) when a user might be busy entering details. Default: `"5m"` (5 minutes). |
```